### PR TITLE
Fix signing in official build with 3rd party libraries

### PR DESCRIPTION
**Pull request by safern**
cc: @akoeplinger @marek-safar @dotnet/runtime-infrastructure

### CODE REVIEW

File: eng/Signing.props [hunk 1]
```diff
@@ -28,6 +28,14 @@
 
       <!-- apphost and comhost template files are not signed, by design. -->
       <FileSignInfo Include="apphost.exe;singlefilehost.exe;comhost.dll" CertificateName="None" />
+
+      <!-- Third-party components which should be signed.  -->
+      <FileSignInfo Include="Newtonsoft.Json.dll" CertificateName="3PartySHA2" />
+      <FileSignInfo Include="AngleSharp.dll" CertificateName="3PartySHA2" />
```
Comment (akoeplinger) [inline eng/Signing.props:34]: this should be removed, it's not something we build/distribute in this repo

**Review by akoeplinger — CHANGES_REQUESTED** [review]

**Review by akoeplinger — APPROVED** [review]

File: eng/Signing.props (hunk 1 above)
Comment (safern) [reply eng/Signing.props]: Haha copy paste from aspnet core and forgot to remove, thanks.

**Comment by safern** [comment]
Merging to unblock official build.

**Pull request merged**